### ISSUE TITLE
Working with Alexa skills

### ISSUE THREAD

**Issue by alexcaulfield** [stdlib/lib#69]
@keithwhor I originally was getting the same error as @cottonsean but then I changed the package.json to use the library used in the hackernoon example. When I tried to run the command lib . /nameofintent/, it gave me this:
![image](https://user-images.githubusercontent.com/19693552/26888310-a42352a2-4b78-11e7-8eb1-20b7a17d2671.png)
However, I was able to move the two files in the main folder and into it's parent's directory, and the command ran smoothly, giving the correct output. However, this would then not allow me to upload to the cloud, and then once I changed the files to be back int he main folder, it would upload but the program wouldn't run. Any chance that the documentation could be updated/there be a new tutorial posted that works with the faaslang spec?

**Comment by jacoblee93** [comment]
Hey @alexcaulfield,

Unfortunately we haven't had a chance to update the Alexa template and our docs yet to our new standard, but it'll be soon! I apologize for the inconvenience -- we'll keep you posted.

Jacob

**Comment by keithwhor** [comment]
Hey @alexcaulfield and @cottonsean -

We've updated our Alexa template as of https://github.com/stdlib/utils.templates/commit/dd6ad787cbf5f38a6f6fffc7ba898bb796b3815c.

We've also updated the Alexa Skills tutorial to match it - https://hackernoon.com/build-an-alexa-skill-in-7-minutes-flat-with-node-js-and-stdlib-70611f58c37f

Let me know if you have any more problems! Try creating from a fresh template. :)
